Dockerfile.template

```diff
@@ -0,0 +1,30 @@
+# install a Debian image with Python 3 installed
+FROM balenalib/%%BALENA_MACHINE_NAME%%-python:3-stretch-run
+
+RUN install_packages apt-utils
+RUN install_packages i2c-tools
+RUN install_packages python3-smbus
+
+# use apt-get if you need to install dependencies
+RUN apt-get update && apt-get install -yq --no-install-recommends \
+	python3-envirophat \
+	libraspberrypi-bin
+
+
+# define our working directory in the container
+WORKDIR /usr/src/app
+
+# copy all our project into /usr/src/app/ folder in the container.
+COPY . ./
+
+# So in our container we should now have:
+# /usr/src/app/
+#           |── Dockerfile
+#           └── project
+#               └── main.py
+
+# load up the i2c module
+CMD modprobe i2c-dev
```

Comment:
this CMD will never run and should be removed. You can only have one `CMD` per dockerfile

Reply:
Do we need to run `modprobe i2c-dev` any more or is this legacy? Seems to deploy fine without it?